libparsec/crates/protocol/tests/authenticated_cmds/v4/shamir_recovery_setup.rs

```diff
@@ -7,33 +7,34 @@ use libparsec_types::{DateTime, InvitationToken};
 
 use super::authenticated_cmds;
 
+// TODO: Fix macro
```

Comment:
@AureliaDolo I had to disable this macro, which has a compilation error that I didn't know how to fix. 

It seems that `$expected:path` cannot be used in `p_assert_q` which expects a value.
```
expected value, found struct variant `authenticated_cmds::shamir_recovery_setup::Rep::ShamirSetupAlreadyExists`
not a valuerustc[Click for full compiler diagnostic](rust-analyzer-diagnostics-view:/diagnostic%20message%20%5B1%5D?1#file%3A%2F%2F%2Fhome%2Fmarcos%2Fcode%2Fscille%2Fparsec-cloud%2Flibparsec%2Fcrates%2Fprotocol%2Ftests%2Fauthenticated_cmds%2Fv4%2Fshamir_recovery_setup.rs)
shamir_recovery_setup.rs(43, 5): Error originated from macro call here
```

```
error[E0308]: mismatched types
   --> libparsec/crates/protocol/tests/authenticated_cmds/v4/shamir_recovery_setup.rs:158:5
    |
158 | /     test_roundtrip_serialization!(
159 | |         "85a6737461747573b974696d657374616d705f6f75745f6f665f62616c6c7061726bbc"
160 | |         "62616c6c7061726b5f636c69656e745f6561726c795f6f6666736574cb404000000000"
161 | |         "0000bb62616c6c7061726b5f636c69656e745f6c6174655f6f6666736574cb40400000"
...   |
165 | |         authenticated_cmds::shamir_recovery_setup::Rep::load
166 | |     );
    | |     ^
    | |     |
    | |_____expected `ShamirRecoverySetupRep`, found fn item
    |       expected because this is `ShamirRecoverySetupRep`
    |
    = note: expected enum `ShamirRecoverySetupRep`
            found fn item `fn() {authenticated_cmds::v4::shamir_recovery_setup::req}`
    = note: this error originates in the macro `$crate::assert_eq` which comes from the expansion of the macro `test_roundtrip_serialization` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Reply:
I've created an issue to fix this later: https://github.com/Scille/parsec-cloud/issues/7687